### PR TITLE
Feature/treatment effect

### PR DESCRIPTION
**Pull request by halfhorst**
fitting the treatment into the risk factor pattern

### CODE REVIEW

File: src/vivarium_csu_ltbi/tools/builder.py [hunk 1]
```diff
@@ -33,7 +33,8 @@ def get_filled_with(self, fill_value):
 
     @staticmethod
     def get_and_package_dismod_ltbi_incidence(loc):
-        datafile = ltbi_paths.get_ltbi_inc_input_artifact_path(loc)
+        datafile = ltbi_paths.get_ltbi_inc_output_artifact_path(loc)
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/tools/builder.py]: old change

File: src/vivarium_csu_ltbi/tools/builder.py [hunk 2]
```diff
@@ -67,7 +68,7 @@ def get_hh_tuberculosis_exposure(loc):
         cat1['parameter'] = 'cat1'
         cat2 = df.copy()
         cat2['parameter'] = 'cat2'
-        cat1['value'] = 1 - cat2['value']
+        cat2['value'] = 1 - cat1['value']
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/tools/builder.py]: old change

File: src/vivarium_csu_ltbi/tools/builder.py [hunk 3]
```diff
@@ -363,18 +364,108 @@ def write_relative_risk_data(art, location):
...
 
     affects_hiv_neg = full_cat_data.copy()
-    affects_hiv_neg['affected_measure'] = 'ltbi_susceptible_hiv_to_activetb_susceptible_hiv'
+    affects_hiv_neg['affected_entity'] = 'ltbi_susceptible_hiv_to_activetb_susceptible_hiv'
 
     full_rr_data = pd.concat([affects_hiv_pos, affects_hiv_neg], axis=0)
     full_rr_data = full_rr_data.set_index(['affected_entity', 'affected_measure'], append=True)
 
-    write(art, 'ltbi_treatment.relative_risk', full_rr_data, skip_interval_processing=True)
+    write(art, 'risk_factor.ltbi_treatment.relative_risk', full_rr_data, skip_interval_processing=True)
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/tools/builder.py]: all old

File: src/vivarium_csu_ltbi/tools/builder.py [hunk 4]
```diff
@@ -513,6 +604,9 @@ def build_ltbi_artifact(loc, output_dir=None):
     write_exposure_risk_data(art, data)
     write_baseline_coverage_levels(art, loc)
     write_adherence_data(art, loc)
-    write_relative_risk_data(art, loc)
+    write_treatment_relative_risk_data(art, loc)
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/tools/builder.py]: old

File: src/vivarium_csu_ltbi/tools/builder.py [hunk 5]
```diff
@@ -363,18 +364,108 @@ def write_relative_risk_data(art, location):
...
     full_rr_data = full_rr_data.set_index(['affected_entity', 'affected_measure'], append=True)
 
-    write(art, 'ltbi_treatment.relative_risk', full_rr_data, skip_interval_processing=True)
+    write(art, 'risk_factor.ltbi_treatment.relative_risk', full_rr_data, skip_interval_processing=True)
+    write(art, 'risk_factor.ltbi_treatment.distribution', 'ordered_polytomous', skip_interval_processing=True)
+
+    
+
+def write_population_attributable_fraction_data(art, location):
+    logger.info(f"Computing population attributable fraction...")
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/tools/builder.py:391]: all old

**Review by collijk — APPROVED** [review]

**Pull request merged**